graphql_compiler/schema_transformation/merge_schemas.py

```diff
@@ -161,6 +138,89 @@ def _get_basic_schema_ast(query_type):
     return blank_ast
 
 
+def _merge_single_schema(merged_schema_ast, name_to_schema_id, scalars, directives,
+                         current_schema_id, current_ast):
+    """Merge current_ast into merged_schema_ast, update all records accordingly.
+
+    Args:
+        merged_schema_ast: Document; modified by this function as current_ast is incorporated
+        name_to_schema_id: Dict[str, str], mapping type name to id of the schema that the
+                           type is from; modified by this function
+        scalars: Set[str], names of all scalars in the merged_schema so far; potentially
```

Comment:
Are you sure your use of ; is grammatically correct?

Reply:
Very possibly it's not grammatical. I'll change the ; to periods and change sentence fragments to full sentences.